src/c4v/scraper/persistency_manager/base_persistency_manager.py

```diff
@@ -0,0 +1,73 @@
+"""
+    Base Persistency Manager: We need a way to handle persistency, 
+    as simple files or maybe with a more complex scheme, such as databases
+    or http requests. This base class provides a contract that
+    every persistency manager should match in order to be used with our app.
+
+    Notice that 
+"""
+
+# Python imports
+from typing import List, Callable
+
+# Local imports
+from c4v.scraper.scraped_data_classes.scraped_data import ScrapedData
+
+
+class BasePersistencyManager:
+    """
+        Base class to provide support for persistency management
+    """
+
+    def get_matching(
```

Comment:
What is the purpose of this API?

This is a search, more concretely, it's equality based search which I don't think we need and it's just too expensive to have.
If we need search capabilities, the API should reflect the boundaries of a realistic search.